providers/implementations/ciphers/cipher_des_hw.c

```diff
@@ -136,7 +136,7 @@ static int cipher_hw_des_cfb1_cipher(PROV_CIPHER_CTX *ctx, unsigned char *out,
 {
     size_t n, chunk = MAXCHUNK / 8;
     DES_key_schedule *key = &(((PROV_DES_CTX *)ctx)->dks.ks);
-    unsigned char c[1], d[1];
+    unsigned char c[1], d[1] = { 0 };
```

Comment:
This may please coverity, but now the line really looks awkward to humans and might cause them to ask questions: why don't you just initialize both variables?

Reply:
Split line for cosmetic purposes.